framework/auth/core.py

```diff
@@ -843,6 +848,22 @@ def _get_unconfirmed_email_for_token(self, token):
 
         return verification['email']
 
+    def verify_token(self, token):
+        """Return whether or not a confirmation token is valid for this user.
+        :rtype: bool
+        """
+        try:
+            verification = self.email_verifications[token]
+        except KeyError:
+            return False
```

Comment:
As suggested in a previous comment: because this method return a boolean, there is no way to distinguish between (1) if `token` is invalid and (2) `token` is expired. 

This method should probably mirror the behavior of `get_confirmation_token`: raise an `ExpiredTokenError` for expired tokens and raise a `KeyError` if the token is not a valid token for the user.